### PR TITLE
Update GetRelatedPlaces API to accomadate to data change

### PR DESCRIPTION
**Pull request by shifucun**
- Remove mmethod, scaling factor and unit from request

### CODE REVIEW

File: proto/mixer.proto [hunk 1]
```diff
@@ -314,21 +314,12 @@ message GetRelatedPlacesRequest {
...
   string measurement_qualifier = 7;
 
-  // (Opitonal) Scaling factor.
-  string scaling_factor = 8;
-
-  // (Optional) Unit.
-  string unit = 9;
-
   // The statistics type, like "measured", "median", etc...
   string stat_type = 10;
```
Comment (Spaceenter) [inline proto/mixer.proto:325]: add 

reserved 5, 8, 9;

?

File: e2etest/get_related_places_test.go [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+			"CriminalActivities",
+			"count",
+			"measuredValue",
+			map[string]string{
+				"crimeType": "UCR_CombinedCrime",
+			},
+		},
+	} {
+		// TODO: remove this when dataset is updated.
+		continue
```
Comment (Spaceenter) [inline e2etest/get_related_places_test.go:95]: Remove this?

File: proto/mixer.proto (hunk 1 above)
Comment (shifucun) [reply proto/mixer.proto]: Added them back for now

File: e2etest/get_related_places_test.go (hunk 2 above)
Comment (shifucun) [reply e2etest/get_related_places_test.go:95]: if remove test won't pass. Need to use the new dataset.

**Pull request merged**